src/tests/test_example_diffs.rs

```diff
@@ -1606,6 +1607,40 @@ src/align.rs:71: impl<'a> Alignment<'a> { │
         assert!(output.contains(r"src/delta.rs: mode -x"));
     }
 
+    #[test]
+    fn test_file_mode_change_unexpected_bits() {
+        let config =
+            integration_test_utils::make_config_from_args(&["--navigate", "--right-arrow=->"]);
+        let output =
+            integration_test_utils::run_delta(GIT_DIFF_FILE_MODE_CHANGE_UNEXPECTED_BITS, &config);
+        let output = strip_ansi_codes(&output);
+        assert!(output.contains(r"Δ src/delta.rs: 100700 -> 100644"));
+    }
```

Comment:
Incidentally, there's a utility to make writing these tests easier now. It doesn't yet have an `expect_contains()` method but I'll add that, at which point this test will be able to be written as

```rust
    fn test_file_mode_change_unexpected_bits() {
        DeltaTest::with(&["--navigate", "--right-arrow=->"])
            .with_input(GIT_DIFF_FILE_MODE_CHANGE_UNEXPECTED_BITS)
            .expect_contains(r"Δ src/delta.rs: 100700 -> 100644");
    }
```

Reply:
Thanks for pointing that out.  I've created a pull request that implements this (along with some other changes).